schema.sql

```diff
@@ -68,4 +63,4 @@ ALTER TABLE animals
 ALTER TABLE animals
     ADD CONSTRAINT fkey_owner
     FOREIGN KEY (owner_id)
-    REFERENCES owners(id);
\ No newline at end of file
+    REFERENCES owners(id);
```

Comment:
- [ ] Please add the created indexes for each foreign key you tested on, as per [this](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_performance_audit.md#:~:text=Any%20changes%20you%20made%20to%20your%20database%20schema%20should%20be%20added%20to%20the%20schema.sql%20file%20and%20commited%20in%20a%20new%20pull%20request.) requirement.